daemon/anonymous-statistics.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env sh
+
+# Valid actions:
+
+# - UPDATE      - updated using netdata-updater.sh
+#      ACTION_RESULT  -- OK or FAILED
+# - FATAL       - netdata exited due to a fatal condition
+#      ACTION_RESULT  -- error code
+#      ACTION_DATA    -- error message
+# - START       - netdata started
+# - INSTALL     - installation action
+#      ACTION_DATA    -- installation method from /var/lib/netdata/.installation
+#      ACTION_RESULT  -- OK or FAILED
+
+ACTION="${1}"
+ACTION_RESULT="${2}"
+ACTION_DATA="${3}"
+
+# -------------------------------------------------------------------------------------------------
+# check opt-out
+
+OPTOUT=
+if [ -z "${NETDATA_CONFIG_DIR}" ]; then
+    for dir in "$(netdata -W get global 'config directory' '/etc/netdata' 2>/dev/null)" "/etc/netdata" "/opt/netdata/etc/netdata"; do
+        if [ -f "${dir}/.opt-out-from-anonymous-statistics" ]; then
+            OPTOUT=all
+            break
+        fi
+    done
+else
+    if [ -f "${NETDATA_CONFIG_DIR}/.opt-out-from-anonymous-statistics" ]; then
+        OPTOUT=all
+    fi
+fi
+
+if [ "${OPTOUT}" = "all" ] ; then
```

Comment:
Check is unnecessary after changing previous assignments of `OPTOUT=all` to `exit 0`